tests/migrations/test_operations.py

```diff
@@ -19,6 +19,10 @@
     sqlparse = None
 
 
+class Mixin(object):
```

Comment:
That's what I originally tried doing but it resulted in a failure on Python 3 as the string representation of the class end up something like `<class 'migrations.test_operations.OperationTestBase.test_create_model_with_duplicate_base.<locals>.Mixin'>`